docs/develop/01_sdk/02_cookbook/06_messaging/01_messaging.md

```diff
@@ -0,0 +1,55 @@
+---
+id: messaging_book
+title: Generate a Message
+---
+
+import TsJsBlock from '@site/src/components/TsJsBlock';
+
+import GenerateRequestCredentialMessage from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/messaging/01_generate_request_credential_message.ts';
+import EncryptMessage from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/messaging/02_encrypt_message.ts';
+import DecryptMessage from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/messaging/03_decrypt_message.ts';
+
+KILT defines a [unicast](https://en.wikipedia.org/wiki/Unicast) [messaging protocol](../../../../concepts/06_messaging.md).
+Each of the messages sent is encrypted using the [DID key agreement key](https://www.w3.org/TR/did-core/#key-agreement).
```

Comment:
At this point, if we want to give details, we should also explain what type of encryption algorithm is used for encryption. I.e., authenticated encryption, i.e., `xsalsa20poly1305`.

Reply:
I suggest that this be linked to a glossary when we have the glossary.

Reply:
Yeah I deleted this comment since I saw it was mentioned below, but something must have gone wrong since it's still here. It can be ignored.